stories/dooboo-ui/Card.stories.tsx

```diff
@@ -0,0 +1,155 @@
+        <Card
+          containerStyle={{ marginBottom: 30 }}
+          image={{
+            uri:
+              'https://gw.alipayobjects.com/zos/rmsportal/JiqGstEfoWAOHiTxclqi.png',
+          }}
+          title="Title"
+          subTitle="subTitle">
+          <Text>{'children'}</Text>
+        </Card>
+        <Title>{'Image'}</Title>
+        <Card
+          containerStyle={{ marginBottom: 30 }}
+          image={{
+            uri:
+              'https://gw.alipayobjects.com/zos/rmsportal/JiqGstEfoWAOHiTxclqi.png',
+          }}
+        />
+        <Title>{'Title'}</Title>
+        <Card containerStyle={{ marginBottom: 30 }} title="Title" />
+        <Title>{'SubTitle'}</Title>
+        <Card containerStyle={{ marginBottom: 30 }} subTitle="subTitle" />
+        <Title>{'Title & SubTitle'}</Title>
+        <Card
+          containerStyle={{ marginBottom: 30 }}
+          title="Title"
+          subTitle="subTitle"
+        />
+        <Title>{'Children'}</Title>
+        <Card containerStyle={{ marginBottom: 30 }}>
+          <Text>{'children'}</Text>
+        </Card>
+        <Title>{'Title & SubTitle & Children'}</Title>
+        <Card
+          containerStyle={{ marginBottom: 30 }}
+          title="Title"
+          subTitle="subTitle">
+          <Text>{'children'}</Text>
+        </Card>
+      </Container>
+    </ScrollContainer>
+  );
+}
+
+function Style(): React.ReactElement {
+  return (
+    <ScrollContainer>
+      <Container>
+        <Title>{'default'}</Title>
+        <Card
+          containerStyle={{ marginBottom: 30 }}
+          image={{
+            uri:
+              'https://gw.alipayobjects.com/zos/rmsportal/JiqGstEfoWAOHiTxclqi.png',
+          }}
+          title="Title"
+          subTitle="subTitle">
+          <Text>{'children'}</Text>
+        </Card>
+        <Title>{'raised'}</Title>
+        <Card
+          containerStyle={{ marginBottom: 30 }}
+          image={{
+            uri:
+              'https://gw.alipayobjects.com/zos/rmsportal/JiqGstEfoWAOHiTxclqi.png',
+          }}
+          title="Title"
+          subTitle="subTitle"
+          raised={true}>
+          <Text>{'children'}</Text>
+        </Card>
+        <Title>{'outlined'}</Title>
+        <Card
+          containerStyle={{ marginBottom: 30 }}
+          image={{
+            uri:
+              'https://gw.alipayobjects.com/zos/rmsportal/JiqGstEfoWAOHiTxclqi.png',
+          }}
+          title="Title"
+          subTitle="subTitle"
+          outlined={true}>
+          <Text>{'children'}</Text>
+        </Card>
+      </Container>
+    </ScrollContainer>
+  );
+}
+
+/**
+ * Below are stories for web
+ */
+export default {
+  title: 'Card',
+};
+
+export const toStorybook = (): ReactElement => <Default />;
+export const toStorybook1 = (): ReactElement => <Style />;
+
+toStorybook.story = {
+  name: 'default',
+};
+
+toStorybook1.story = {
+  name: 'Style',
+};
+/**
```

Comment:
break the line.

Reply:
I'd like you to explain a little more.

Reply:
It needs a line break between 140 and 141 line number.

Reply:
Thank you! I understood and corrected.